applications/asset_tracker_v2/src/modules/Kconfig.debug_module

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DEBUG_MODULE
+	bool "Debug module"
+
+if DEBUG_MODULE
+
+config DEBUG_MODULE_USE_CUSTOM_TRANSPORT
+	bool "Use custom transport when sending memfault data"
```

Comment:
Should we not have a "default" value here?

Reply:
Kconfig options automatically defaults to "n". So adding `default n` is not needed.